### PR TITLE
Pislaru daniel login test

### PR DESCRIPTION
**Pull request by S4veZone**
Add LoginTest for all UsersName

### CODE REVIEW

File: src/test/java/e2e/LoginTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package e2e;
+
+import e2e.pages.LoginPage;
+import org.testng.annotations.Test;
+
+public class LoginTest extends TestBase {
+    LoginPage loginPage;
+
+    @Test
+    public void userCanLogin(){
```
Comment (LeonadO555) [inline src/test/java/e2e/LoginTest.java:10]: standartUserCanLogin

File: src/test/java/e2e/LoginTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        String email = "standard_user";
+        String password = "secret_sauce";
+
+        loginPage = new LoginPage(app.driver);
+        loginPage.login(email, password);
+
+    }
+    @Test
+    public void userCanLogin1(){
```
Comment (LeonadO555) [inline src/test/java/e2e/LoginTest.java:19]: problemUserCanLogin

File: src/test/java/e2e/LoginTest.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        String email = "problem_user";
+        String password = "secret_sauce";
+
+        loginPage = new LoginPage(app.driver);
+        loginPage.login(email, password);
+
+    }
+    @Test
+    public void userCanLogin2(){
```
Comment (LeonadO555) [inline src/test/java/e2e/LoginTest.java:28]: performanceGlitchUserCanLogin

File: src/test/java/e2e/LoginTest.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        String email = "performance_glitch_user";
+        String password = "secret_sauce";
+
+        loginPage = new LoginPage(app.driver);
+        loginPage.login(email, password);
+
+    }
+    @Test
+    public void userCanLogin3(){
+        String email = "error_user";
```
Comment (LeonadO555) [inline src/test/java/e2e/LoginTest.java:38]: errorUserCanLogin

File: src/test/java/e2e/LoginTest.java [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        String password = "secret_sauce";
+
+        loginPage = new LoginPage(app.driver);
+        loginPage.login(email, password);
+
+    }
+    @Test
+    public void userCanLogin4(){
```
Comment (LeonadO555) [inline src/test/java/e2e/LoginTest.java:46]: visualUserCanLogin

File: src/test/java/e2e/LoginTest.java [hunk 6]
```diff
@@ -0,0 +1,56 @@
+package e2e;
+
+import e2e.pages.LoginPage;
+import org.testng.annotations.Test;
+
+public class LoginTest extends TestBase {
```
Comment (LeonadO555) [inline src/test/java/e2e/LoginTest.java:6]: Your all tests haven't check that user logged in
Need to add check that user logged in for every test

**Review by LeonadO555 — APPROVED** [review]

File: src/test/java/e2e/LoginTest.java (hunk 6 above)
Comment (S4veZone) [reply src/test/java/e2e/LoginTest.java:6]: Ok I Understand. , make this after make next page

**Pull request merged**